File: VisitBosnia/VisitBosnia.Model/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model
{
    public class Post
    {
        public int Id { get; set; }
        public int AppUserId { get; set; }
        public int ForumId { get; set; }
        public DateTime CreatedTime { get; set; }
        public string Title { get; set; } = null!;
        public string Content { get; set; } = null!;

        public virtual AppUser AppUser { get; set; } = null!;
        public virtual Forum Forum { get; set; } = null!;
    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/post/post.dart
import 'package:visit_bosnia_mobile/model/appUser/app_user.dart';

class Post {
  int? id;
  int? appUserId;
  int? forumId;
  String? createdTime;
  String? title;
  String? content;
  AppUser? appUser;
  // Null? forum;

  Post({
    this.id,
    this.appUserId,
    this.forumId,
    this.createdTime,
    this.title,
    this.content,
    this.appUser,
    // this.forum
  });

  Post.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    appUserId = json['appUserId'];
    forumId = json['forumId'];
    createdTime = json['createdTime'];
    title = json['title'];
    content = json['content'];
    appUser =
        json['appUser'] != null ? new AppUser.fromJson(json['appUser']) : null;
    // appUser = json['appUser'];
    // forum = json['forum'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['appUserId'] = this.appUserId;
    data['forumId'] = this.forumId;
    data['createdTime'] = this.createdTime;
    data['title'] = this.title;
    data['content'] = this.content;
    if (this.appUser != null) {
      data['appUser'] = this.appUser!.toJson();
    }
    // data['appUser'] = this.appUser;
    // data['forum'] = this.forum;
    return data;
  }
}
